versions.props

```diff
@@ -19,7 +19,6 @@ commons-io:commons-io = 2.1
 io.atomix:* = 1.0.0-rc9
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
-io.dropwizard.metrics:metrics-* = 3.1.1
```

Comment:
We should be explicit in the `com.codahale.metrics:metrics-*` versions here since we're trying to keep the Java 6 compatible version for now.